src/__tests__/api.test.js

```diff
@@ -0,0 +1,8 @@
+import store from '../redux/configureStore';
+
+describe('Countries redux state tests', () => {
+  it('Should initially set countries to an empty array', () => {
+    const state = store.getState().characters;
+    expect(state.characters).toEqual([]);
+  });
+});
```

Comment:
- [ ] [optional] It would have been nice if you added more tests for your redux methods.

Reply:
I highlited sssska
I think u need to wear your glasses

Reply:
[optional changes] @NurkaAmre I just suggested more tests I already saw you have added a test, also if possible it would be nice to remove one readme file.